src/data/faq.json

```diff
@@ -188,12 +188,11 @@
                             "<ul><li>the respective laboratory is not fully linked with the Corona Warn App backend.</li><li>the consent field allowing the transmission of the test result to the Corona-Warn-App was not ticked on the submission form (10C/OEGD) sent by your doctor to the laboratory.</li></ul>",
                             "Please check whether the laboratory working with your doctor or test center is connected to the Corona-Warn-App and whether the consent field is checked on the form. Only then is it ensured that the procedure used works with the Corona-Warn-App.",
                             "Currently, 90% of the existing test capacities in Germany are covered by the complete digital process. We are continuing to work with the laboratories to close the gaps and to connect and upgrade all laboratories for the process. Therefore it can happen that a laboratory gets the order for the test with the correct form, but has to transmit the test result to the tested person in another way.",
-                            "In this case, please call the responsible medical staff or test center/health authority to get your test result. If you are informed of a positive test result, you can request a TeleTAN by calling the TAN hotline on 0800 7540002 to warn your contacts via the Corona-Warn-App. Before you are able to enter the TeleTAN into the app, you have to delete the pending test result from the app. To do this, please tap on",
-                            "<ul><li>'Display Test' -> 'Delete test</li></ul>",
+                            "In this case, please call the responsible medical staff or test center/health authority to get your test result. If you are informed of a positive test result, you can request a TeleTAN from the TAN Hotline (<a href='#international_phone_numbers' target='_blank' rel='noopener noreferrer'>click here to see our phone numbers</a>) to warn your contacts via the Corona-Warn-App. Before you are able to enter the TeleTAN into the app, you have to delete the pending test result from the app. To do this, please tap on",

```

Comment:
The link to #international_phone_numbers only gives the TAN hotline +49 30 498 75402 which is not free-of-charge like the 0800 7540002 number which was previously listed.

Reply:
I had the same thought but there's a PR for this: #1078

Reply:
yes, they will all link to a general hotline article with national/international numbers